app/models/school_cohort.rb

```diff
@@ -59,10 +59,22 @@ def lead_provider
     school.lead_provider(cohort.start_year)
   end
 
+  def previous_lead_provider
+    school.lead_provider(cohort.start_year - 1)
+  end
+
   def delivery_partner
     school.delivery_partner_for(cohort.start_year)
   end
 
+  def delivery_partner_to_be_confirmed?
+    default_induction_programme.delivery_partner_to_be_confirmed
+  end
+
+  def previous_delivery_partner
+    school.delivery_partner_for(cohort.start_year - 1)
+  end
+
```

Comment:
Same as above, can we move this somewhere else or delete it please if it's not used?

Reply:
not used and removed